app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -91,7 +79,7 @@
             = _("Required")
           %span.help-block{"ng-show" => "!angularForm.#{prefix}_verify.$error.required && #{prefix}_VerifyCtrl != undefined && #{prefix}_VerifyCtrl.$error.verifypasswd"}
             = passwd_mismatch
-    %div{"ng-show" => "showVerify('#{prefix}_userid')"}
+    %div{"ng-show" => "showValidate('#{prefix}')"}
```

Comment:
@AparnaKarve, can you look this over?  This looks like a change to the way all providers would decide to show the validate credentials button.

See also [the controller.js changes above](https://github.com/ManageIQ/manageiq/pull/8540/files#diff-791601547dc3422acff81403f91fb6ffR51).